include/sot-dynamic-pinocchio/dynamic-pinocchio.h

```diff
@@ -103,7 +103,7 @@ class SOTDYNAMIC_EXPORT DynamicPinocchio
     public:
   /* --- SIGNAL ACTIVATION --- */
   dg::SignalTimeDependent< dg::Matrix,int >&
-    createEndeffJacobianSignal( const std::string& signame, const std::string& );
+    createEndeffJacobianSignal( const std::string& signame, const std::string&, const bool isLocal);
```

Comment:
You can maybe use `const bool isLocal = true`, this way your modification will not risk to break anything relying on this API.